backend/modules/pdf_processing/ai_extraction_service.py

```
@@ -261,49 +167,67 @@ async def extract_personas_and_key_figures(
+- Default to "Business Analyst" if no specific protagonist is identified
+
+━━━ PERSONA FIELD GUIDE ━━━
+For each key figure, populate all fields using ONLY information from the case study:
+
+• name: Full name and title as stated in the case
+• role: Their position/title
+• background: Professional history, experience, and context within the organization (2-3 sentences)
+• current_context: Their current responsibilities, specific challenges, and perspective as they relate to the case events (2-3 sentences — distinct from background)
+• correlation: How this persona relates to the student (protagonist) role
+• personality_traits: Big Five model, scored 1 (lowest) to 10 (highest) based on how the case portrays this person
+• primary_goals: 3–5 concise, specific goals this persona is actively pursuing in the simulation
```

Ruff (0.15.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)